packages/lib/package.json

```diff
@@ -6,8 +6,7 @@
   "private": false,
   "license": "MIT",
   "scripts": {
-    "test": "./scripts/test.sh",
-    "test-contracts": "TS_NODE_PROJECT='tsconfig.test.json' npx mocha --exit --require ts-node/register --recursive test/contracts",
+    "test": "TS_NODE_PROJECT='tsconfig.test.json' npx mocha --exit --timeout 5000 --require ts-node/register --recursive test",
```

Comment:
Can we set the timeout explicitly only for the tests that we _know_ are slow, instead of for the entire suite?

Reply:
Turns out only a single test required it, and mocha provides a handy way of configuring it by calling `.timeout` on the return value of `it`:

```javascript
it('is slow', () => {
}).timeout(10000);
```